web/app/view/ReportController.js

```diff
@@ -125,6 +128,7 @@ Ext.define('Traccar.view.ReportController', {
                 } else {
                     store = this.getGrid().getStore();
                 }
+                store.showMarkers = this.showMarkers;
```

Comment:
Agree with you. But I know only two ways of passing  values between controllers.
- `getCmp`
- events

Events do not fit, `getCmp` is very undesirable.